.github/workflows/ruby-build.yml

```diff
@@ -116,21 +116,21 @@ jobs:
           key: ruby-build
       - name: Build Query Pack
         run: |
-          rm -rf target/packs
-          codeql pack create ../misc/suite-helpers --output target/packs
-          codeql pack create ../shared/regex --output target/packs
-          codeql pack create ../shared/ssa --output target/packs
-          codeql pack create ../shared/tutorial --output target/packs
-          codeql pack create ql/lib --output target/packs
-          codeql pack create -j0 ql/src --output target/packs --compilation-cache "${{ steps.query-cache.outputs.cache-dir }}"
-          PACK_FOLDER=$(readlink -f target/packs/codeql/ruby-queries/*)
+          rm -rf query-packs
```

Comment:
The reason for using the `target` folder is to avoid interference with any folder that may be present in the repository. We don't have a `query-packs` folder, so it is fine for now, but we create one in the future,

Reply:
Using `${{ runner.temp }}` is probably best. That is like `/tmp` but for Actions, and also works on Windows.